docs/getting-started.md

```diff
@@ -4,9 +4,9 @@ title: Getting Started
 sidebar_label: Getting Started
 ---
 
-Getting started on Polkadot is easy! The wiki already contains guides for doing most of the core
-functionality. Our wiki has three main sections: Learn (for learners), Build (for people that want
-to build on Polkadot), and Maintain (for people that want to maintain the network).
+Welcome to the Polkadot Wiki! This wiki contains guides for interacting with the core functionality.
+Our wiki has three main sections: Learn (for learners), Build (for people that want to build on
```

Comment:
`wiki` said 3 times in 2 lines, maybe reduce.

Reply:
Good point there, actually!

Reply:
Does "this central source of truth" sound better or a bit worse? :)

Reply:
Haha, better I think!